Lesson4.ipynb

```diff
@@ -29,6 +33,737 @@
+      "Total steps of 5-4 is:256992\n",
+      "Total steps of 5-5 is:358159\n",
+      "Total steps of 6-1 is:271712\n",
+      "Total steps of 6-2 is:336492\n",
+      "Total steps of 6-3 is:268814\n",
+      "Total steps of 6-4 is:255806\n",
+      "Total steps of 6-5 is:201153\n",
+      "Total steps of 7-1 is:266423\n",
+      "Total steps of 7-2 is:198659\n",
+      "Total steps of 7-3 is:113853\n",
+      "Total steps of 7-4 is:276236\n",
+      "Total steps of 7-5 is:252171\n",
+      "Total steps of 8-1 is:214189\n",
+      "Total steps of 8-2 is:291179\n",
+      "Total steps of 8-3 is:424085\n",
+      "Total steps of 8-4 is:285982\n",
+      "Total steps of 8-5 is:269320\n",
+      "Total steps of 9-1 is:164322\n",
+      "Total steps of 9-2 is:160653\n",
+      "Total steps of 9-3 is:335682\n",
+      "Total steps of 9-4 is:209728\n",
+      "Total steps of 9-5 is:258475\n",
+      "Total steps of 10-1 is:322175\n",
+      "Total steps of 10-2 is:307921\n",
+      "Total steps of 10-3 is:162633\n",
+      "Total steps of 10-4 is:193488\n",
+      "Total steps of 10-5 is:172515\n",
+      "Total steps of 11-1 is:188957\n",
+      "Total steps of 11-2 is:322773\n",
+      "Total steps of 11-3 is:289775\n",
+      "Total steps of 11-4 is:240985\n",
+      "Total steps of 11-5 is:353059\n",
+      "Total steps of 12-1 is:283606\n",
+      "Total steps of 12-2 is:296021\n",
+      "Total steps of 12-3 is:316362\n",
+      "Total steps of 12-4 is:196861\n",
+      "Total steps of 12-5 is:387477\n",
+      "Total steps of 13-1 is:200552\n",
+      "Total steps of 13-2 is:197572\n",
+      "Total steps of 13-3 is:155611\n",
+      "Total steps of 13-4 is:297900\n",
+      "Total steps of 13-5 is:203646\n",
+      "Total steps of 14-1 is:331675\n",
+      "Total steps of 14-2 is:289087\n",
+      "Total steps of 14-3 is:140621\n",
+      "Total steps of 14-4 is:70242\n",
+      "Total steps of 14-5 is:108292\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print total steps per person by using DictReader\n",
+    "import csv\n",
+    "with open('Biggest Loser 2018.csv') as csvfile:\n",
+    "    readCSV = csv.DictReader(csvfile)\n",
+    "    header = readCSV.fieldnames\n",
+    "    for row in readCSV:\n",
+    "        totalSteps = 0\n",
+    "        for column_index, steps_in_column in enumerate(header):\n",
+    "            steps_in_column = row[header[column_index]]\n",
+    "            if column_index > 3 and steps_in_column != '':\n",
+    "                totalSteps = totalSteps + int(steps_in_column)\n",
+    "        print(\"Total steps of \" + row['team_member'] + \" is:\" + str(totalSteps))\n",
+    "            "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Write a function for counting total steps\n",
+    "\n",
+    "Steps start from column 5, we can count the total steps by looping header index starts from 4, which is column_index > 3\n",
```

Comment:
"we can count the total steps by looping header index starts from 4" should read "we can count the total steps by looping header index starting from 4"